docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -23,3 +23,48 @@ The steps you need to take to migrate from Material UI v6 to v7 are described
 This list is a work in progress.
 Expect updates as new breaking changes are introduced.
 :::
+
+### Package layout
+
+The package layout has been updated to use the Node.js exports field. This brings several changes:
+
+Deep imports with more than one level are no longer allowed. For example:
+
+```diff
+- import createTheme from '@mui/material/styles/createTheme';
++ import { createTheme } from '@mui/material/styles';
+```
+
+This was never officially supported, but will now be restricted by bundlers and runtimes.
+
+To use the modern bundle (which excludes legacy browser support for smaller bundle size), you'll need to configure your bundler to use the "mui-modern" exports condition:
+
+```js
+// webpack.config.js
+{
+  resolve: {
+    conditionNames: ['mui-modern', '...'],
+  }
+}
+
+// vite.config.js
+{
+  resolve: {
+    conditions: ['mui-modern', 'module', 'browser', 'development|production']
+  }
+}
+```
+
+If you were using a Vite alias to force ESM imports for the icons package, you should remove it as it's no longer necessary:
+
+```diff
+ // vite.config.js
+   resolve: {
+     alias: [
+-      {
```

Comment:
At some point I think we had a suggestion for this for Next.js as well, does the same logic apply?

Reply:
I don't think so, the issue this solved only applies to vite. maybe there was another issue I'm missing?

Reply:
Ah sorry, I confused it with the `modularizeImports` option in Next.js.